script/chart_sync_utils.sh

```diff
@@ -109,22 +108,21 @@ replaceImage_latestToProduction() {
         curl_opts=(-s -H "Authorization: token ${GITHUB_TOKEN}")
     fi
 
-    # Get the latest tag from the bitnami repository
-    # See https://github.com/vmware-tanzu/kubeapps/issues/6220
-    # We may need to update to use dockerhub if we want the most recent tag
-    # explicitly. For now, just create the PR with the latest tag.
-    # local tag
-    # tag=$(curl "${curl_opts[@]}" "https://api.github.com/repos/bitnami/${repoName}/tags" | jq -r '.[0].name')
+    # Get the latest tag from the bitnami repository in DockerHub.
+    # Assumption: the more recent tag is the second one in the list after a reverse alphabetical sorting,
+    # first one is "latest", which we don't actually want.
+    local tag
+    tag=$(curl "${curl_opts[@]}" "https://hub.docker.com/v2/namespaces/bitnami/repositories/${repoName}/tags" | jq -r '.results[].name' | sort -r --version-sort | sed -n "2 {p;q}")
```

Comment:
Nice - I'd not looked at switching from the non-existent GH repos to checking the dockerhub repo instead.